### ISSUE TITLE
Requested items are sometimes not transtported. [1.10.2] [1.12.2]

### ISSUE THREAD

**Issue by Psy-Virus** [MrTJP/ProjectRed#1398]
Hey there,
sometimes it happens that when you request something that is a bit further away, that it doesn't get transported. Instead, when you request it, it vanishes from the request pipe list, but stays in the chest and shows up again when you press refresh. You can work around this by destroying a pipe in the network and rebuild this pipe.

it's build as follows:

+ The main storage:
![2018-09-11_17 25 39](https://user-images.githubusercontent.com/6222200/45370417-81d41480-b5e8-11e8-970e-683138943954.png)
![2018-09-11_17 25 51](https://user-images.githubusercontent.com/6222200/45370452-944e4e00-b5e8-11e8-95e7-016939cf98aa.png)

+ The connection between the bugging Request Pipe and the storage:
There's a junction pipe at about each 40 blocks.
![2018-09-11_17 35 03](https://user-images.githubusercontent.com/6222200/45370732-1c345800-b5e9-11e8-8105-e24b21dc8d14.png)

The chunks should have been active.

Thanks in advance.

**Comment by Psy-Virus** [comment]
Did some more testing. It seems like this happens when the Requestpipe is too far away from the chest, the stuff is stored in.
If one player is requesting for example 1 stack cobble and someone else is looking into the chest where it is stored, player 2 can see that it is not getting transported. Sometimes it vanishes for a split second on request and gets restored.

**Comment by Psy-Virus** [comment]
The count of those items is often also not correct. For stuff that is stored far away you mostly see much more items in the request pipe and you can request them. But they either get transported and not everything is arriving or it looks like it gets transported but on a refresh everything is there again.

**Comment by MrTJP** [comment]
Can you verify that by 'far away' you are still maintaining the max search length of each routed pipe? It is a config option.

**Comment by Psy-Virus** [comment]
Yes. I configured it to 200 and all routing pipes are within a +-50 block distance from each other.

**Comment by Psy-Virus** [comment]
And it also still happens that items get stuck. It feels like not all routing pipes are known by the network.
The problem is that this doesn't happen reproducible. Sometimes it's after you started the server, sometimes it happens later.
